### ISSUE TITLE
TypeError Ctrl+C mocha run in parallel mode

### ISSUE THREAD

**Issue by gomaloick** [josdejong/workerpool#326]
Hello everyone,

I have a TypeError when killing a mocha run in parallel mode with Ctrl+C (windows terminal)
```
node_modules\workerpool\src\WorkerHandler.js:434
              me.worker.killed = true;
                               ^

TypeError: Cannot set property 'killed' of null
```

**Comment by josdejong** [comment]
Thanks for reporting, will be fixed via a9b74d6 (fix not yet published)

**Comment by josdejong** [comment]
The fix is published now in `v6.2.0`
